platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PcesWriter.java

```diff
@@ -34,6 +37,8 @@
 
 /**
  * This object is responsible for writing events to the database.
+ * <p>
+ * Future work: This class will be deleted once the PCES migration to the new framework is complete.
```

Comment:
This `Future work` applied to the old version of this file. The version here will not be deleted

Reply:
removed